Project.toml

```
@@ -64,6 +66,7 @@ Bijections = "0.1, 0.2"
 CommonWorldInvalidations = "1"
 ComponentArrays = "0.15"
 ConstructionBase = "1.2"
+D3Trees = "0.3"
```

```suggestion
D3Trees = "0.3.5"
```

Need D3Trees v0.3.5 (see below).